package.json

```diff
@@ -1,62 +1,52 @@
-    "push-tags": "pnpm exec changeset tag && git pull && git push --follow-tags",
-    "lint": "eslint \"./packages/**/*{.ts,.tsx}\"",
-    "format": "prettier --write .",
-    "postinstall": "husky install"
-  },
-  "devDependencies": {
-    "@changesets/cli": "^2.26.0",
-    "@types/jest": "^29.5.12",
-    "@types/picomatch": "^3.0.0",
-    "@typescript-eslint/eslint-plugin": "^5.60.0",
-    "@typescript-eslint/parser": "^5.60.0",
-    "cypress": "13.13.1",
-    "eslint": "^8.43.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-plugin-react": "^7.32.2",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "husky": "^7.0.4",
-    "picomatch": "^4.0.2",
-    "prettier": "^2.8.4",
-    "pretty-quick": "^3.1.3",
-    "turbo": "^1.8.5",
-    "typescript": "4.6.4",
-    "vite": "^4.3.9",
-    "workspace": "^0.0.1-preview.1"
-  },
-  "pnpm": {
-    "overrides": {
-      "fast-xml-parser": "4.2.5",
-      "json5": "1.0.2",
-      "loader-utils@<1.4.1": ">=1.4.1",
-      "semver": "7.5.2",
-      "ua-parser-js": "0.7.33"    
-    }
-  },
-  "packageManager": "pnpm@9.4.0"
+	"name": "root",
+	"private": true,
+	"version": "0.0.0",
+	"workspaces": [
+		"packages/@tinacms/*",
+		"examples/*",
+		"experimental-examples/*",
+		"packages/[^@]*"
+	],
+	"scripts": {
+		"dev": "pnpm watch",
+		"watch": "pnpm run build && pnpm --filter=\"./packages/@tinacms/scripts\" run watch",
+		"build": "turbo run build --filter=\"./packages/**\"",
+		"test": "turbo run test --filter=\"./packages/**\" -- --silent",
+		"types": "turbo run types --filter=\"./packages/**\"",
+		"build:e2e": "pnpm run build && pnpm --filter=kitchen-sink-starter run build",
+		"start:e2e": "pnpm --filter=kitchen-sink-starter run start ",
+		"start:e2e:dev": "pnpm --filter=kitchen-sink-starter run dev",
+		"test:dev": "pnpm cypress open",
+		"test:e2e": "pnpm cypress run",
+		"version": "pnpm exec changeset version",
+		"version:snapshot": "pnpm exec changeset version --snapshot",
+		"publish": "pnpm publish -r --no-git-checks --access public",
+		"publish:beta": "pnpm publish -r --tag beta --no-git-checks --access public",
+		"publish:dev": "pnpm publish -r --tag dev --no-git-checks --access public",
+		"push-tags": "pnpm exec changeset tag && git pull && git push --follow-tags",
+		"lint": "eslint \"./packages/**/*{.ts,.tsx}\"",
+		"format": "prettier --write .",
+		"postinstall": "husky install"
+	},
+	"devDependencies": {
+		"@changesets/cli": "^2.26.0",
+		"@types/jest": "^29.5.12",
+		"@types/picomatch": "^3.0.0",
+		"@typescript-eslint/eslint-plugin": "^5.60.0",
+		"@typescript-eslint/parser": "^5.60.0",
+		"cypress": "13.13.1",
+		"eslint": "^8.43.0",
+		"eslint-config-prettier": "^8.8.0",
+		"eslint-plugin-react": "^7.32.2",
+		"eslint-plugin-react-hooks": "^4.6.0",
+		"husky": "^7.0.4",
+		"picomatch": "^4.0.2",
+		"prettier": "^2.8.4",
+		"pretty-quick": "^3.1.3",
+		"turbo": "^1.8.5",
+		"typescript": "4.6.4",
```

Comment:
**question**: Should the Typescript version here be updated to 5.5.4 like everywhere else?

Reply:
Yes! 😲